### DISCUSSION TITLE
Cave config?

### DISCUSSION THREAD

**Discussion by PrincessKumico** [yungnickyoung/Better-Caves#151] [Ask Questions]
I made my own modpack and added a bunch of mods I wanted to use. I wanted to remove caves and I did it in ATM6 by going to mods, going to yung's better caves, and clicking config. There it had a really good way to change stuff within the actual game itself. But on the modpack I made I can't seem to do that. The config button is grayed out and can't be clicked? Is there a mod that enables the ability to easily config mods within minecraft that I'm not seeing or?

**Comment by yungnickyoung** [comment]
You'll have to edit the config file manually. You can find it in your minecraft config directory.

If you have any further questions, try asking on the [Discord](https://discord.com/invite/rns3beq) btw, I'm much more active there.
